partner_contact_in_several_companies/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Contacts in several partners",
+    "summary": "Allow to have one contact in several partners",
+    "version": "11.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "https://odoo-community.org/",
+    "author": "Nicolas JEUDY,Odoo Community Association (OCA),Odoo SA",
+    "license": "AGPL-3",
+    'application': False,
```

Comment:
you can remove application, installable and auto_install

Reply:
done